app/controllers/users_controller.rb

```diff
@@ -27,9 +31,12 @@ def edit
   end
 
   # PATCH/PUT /users/1 or /users/1.json
+  # rubocop:disable Metrics/MethodLength
```

Comment:
I think this is no longer needed.  Thanks for breaking up the routine...  Sorry if you already planned to remove it.

Reply:
I am sorry, https://github.com/pulibrary/pdc_describe/pull/574/commits/a542b7eba7d814f2d3d6855c88ad4238c6058718 still raises a failure (please see https://app.circleci.com/pipelines/github/pulibrary/pdc_describe/1738/workflows/7eeffb68-411d-457c-8717-6d95d478a97f/jobs/5069). I am going to restore this.